RELEASE_NOTE.md

```diff
@@ -1,5 +1,27 @@
 # OpenPAI Release Note
 
+## May 2020 (version 1.0.0)
+
+There are **breaking changes** since OpenPAI v1.0.0. Before v1.0.0, OpenPAI was based on Yarn and Kubernetes. Now, OpenPAI has switched to a pure Kubernetes-based architecture. Also, we have created 5 separate repositories for individual OpenPAI components. Some of them are newly-introduced, and some are migrated from [OpenPAI main repository](https://github.com/microsoft/pai):
```

Comment:
Let's move this to later sections (upgrade section). This is not important to people who are don't need to upgrade OpenPAI from yarn to k8s.